server/chat-commands/core.ts

```diff
@@ -1704,12 +1704,14 @@ export const commands: ChatCommands = {
 			let group = targetUser.tempGroup;
 			if (targetUser.locked) group = Config.punishgroups?.locked?.symbol ?? '\u203d';
 			if (targetUser.namelocked) group = Config.punishgroups?.namelocked?.symbol ?? '✖';
+			const sectionleader = Users.globalAuth.sectionLeaders.has(targetUser.id);
 			const userdetails: AnyObject = {
 				id: target,
 				userid: targetUser.id,
 				name: targetUser.name,
 				avatar: targetUser.avatar,
 				group: group,
+				sectionleader,
```

Comment:
```suggestion
			const userdetails: AnyObject = {
				id: target,
				userid: targetUser.id,
				name: targetUser.name,
				avatar: targetUser.avatar,
				group: group,
				sectionleader: Users.globalAuth.sectionLeaders.has(targetUser.id),
```

Reply:
I wonder if this should be something like:

```ts
customgroup: Users.globalAuth.sectionLeaders.has(targetUser.id) ? "Section Leader" : undefined,
```

Reply:
(Similar to spotlight, it would let side servers set their own messages in the group section.)